files/default/sql/ddl/updates/0.10.0.sql

```diff
@@ -4,4 +4,29 @@ DROP TABLE IF EXISTS  `hopsworks`.`projectgenericuser_certs`;
 DROP TABLE IF EXISTS  `hopsworks`.`zeppelin_interpreter_confs`;
 
 -- Remove Zeppelin as service from existing projects
-DELETE FROM `hopsworks`.`project_services` where `service`='ZEPPELIN';
\ No newline at end of file
+DELETE FROM `hopsworks`.`project_services` where `service`='ZEPPELIN';
+
+DROP TABLE IF EXISTS `jupyter_interpreter`;
+
+ALTER TABLE `jupyter_settings` DROP COLUMN `num_tf_ps`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `num_tf_gpus`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `num_mpi_np`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `appmaster_cores`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `appmaster_memory`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `num_executors`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `num_executor_cores`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `executor_memory`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `dynamic_initial_executors`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `dynamic_min_executors`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `dynamic_max_executors`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `mode`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `archives`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `jars`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `files`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `py_files`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `spark_params`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `fault_tolerant`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `umask`;
+ALTER TABLE `jupyter_settings` DROP COLUMN `log_level`;
+ALTER TABLE `jupyter_settings` ADD COLUMN `base_dir` VARCHAR(255) DEFAULT '/Jupyter/';
+ALTER TABLE `jupyter_settings` ADD COLUMN `json_config` TEXT NOT NULL;
```

Comment:
mmm, now sure here. Shouldn't you set the default to be emtpy string to handle already existing `jupyter_settings` entries?
Does the code handle empty settings, or should we write a migration step in Expat?

Reply:
The code will handle it by setting the correct json_config so no need for a default value here. Particulary since it needs to be JSON of the class SparkJobConfiguration.

Reply:
No need for expat at all.

Reply:
Yeah, my bad. I was worried that MySQL would have complained. I ran the query on a VM and it worked fine.